### PR TITLE
fixed some typos and grammatical mistakes

### PR DESCRIPTION
**Pull request by shahincsejnu**

### CODE REVIEW

File: content/en/docs/reference/using-api/api-concepts.md [hunk 1]
```diff
@@ -130,9 +130,9 @@ To retrieve a single list in chunks, two new parameters `limit` and `continue` a
 
 Like a watch operation, a `continue` token will expire after a short amount of time (by default 5 minutes) and return a `410 Gone` if more results cannot be returned. In this case, the client will need to start from the beginning or omit the `limit` parameter.
 
-For example, if there are 1,253 pods on the cluster and the client wants to receive chunks of 500 pods at a time, they would request those chunks as follows:
+For example, if there are 1,253 pods on the cluster, and the client wants to receive chunks of 500 pods at a time, they would request those chunks as follows:
```
Comment (irvifa) [inline content/en/docs/reference/using-api/api-concepts.md:133]:
```suggestion
For example, if there are 1,253 pods on the cluster then the client wants to receive chunks of 500 pods at a time, they would request those chunks as follows:
```

File: content/en/docs/reference/access-authn-authz/authentication.md [hunk 2]
```diff
@@ -233,7 +235,7 @@ secrets:
 - name: jenkins-token-1yvwg
 ```
 
-The created secret holds the public CA of the API server and a signed JSON Web
+The created secret holds the public CA of the API server, and a signed JSON Web
```
Comment (irvifa) [inline content/en/docs/reference/access-authn-authz/authentication.md:238]:
```suggestion
The created secret holds the public CA of the API server and a signed JSON Web
```

**Review by irvifa — CHANGES_REQUESTED** [review]

File: content/en/docs/reference/using-api/api-concepts.md (hunk 1 above)
Comment (kbhawkey) [reply content/en/docs/reference/using-api/api-concepts.md:133]: Hi @shahincsejnu .
The original text seems correct.

File: content/en/docs/reference/access-authn-authz/authentication.md [hunk 3]
```diff
@@ -205,8 +205,10 @@ spec:
 ```
 
 Service account bearer tokens are perfectly valid to use outside the cluster and
-can be used to create identities for long standing jobs that wish to talk to the
-Kubernetes API. To manually create a service account, use the `kubectl
+
+can be used to create identities for long-standing jobs that wish to talk to the
+Kubernetes API. To manually create a service account, simply use the `kubectl
```
Comment (kbhawkey) [inline content/en/docs/reference/access-authn-authz/authentication.md:210]: This change is not needed.

File: content/en/docs/concepts/security/controlling-access.md [hunk 4]
```diff
@@ -101,7 +101,7 @@ If Bob makes a request to write (`create` or `update`) to the objects in the `pr
...
 
-Kubernetes supports multiple authorization modules, such as ABAC mode, RBAC Mode, and Webhook mode. When an administrator creates a cluster, they configure the authorization modules that should be used in the API server. If more than one authorization modules are configured, Kubernetes checks each module, and if any module authorizes the request, then the request can proceed. If all of the modules deny the request, then the request is denied (HTTP status code 403).
+Kubernetes supports multiple authorization modules, such as ABAC mode, RBAC Mode, and Webhook mode. When an administrator creates a cluster, they configure the authorization modules that should be used in the API server. If more than one authorization modules are configured, Kubernetes checks each module, and if any module authorizes the request, then the request can proceed. If all the modules deny the request, then the request is denied (HTTP status code 403).
```
Comment (kbhawkey) [inline content/en/docs/concepts/security/controlling-access.md:104]: ?

File: content/en/docs/reference/access-authn-authz/authentication.md (hunk 3 above)
Comment (irvifa) [reply content/en/docs/reference/access-authn-authz/authentication.md:210]: This changes is not needed.

File: content/en/docs/concepts/security/controlling-access.md (hunk 4 above)
Comment (irvifa) [reply content/en/docs/concepts/security/controlling-access.md:104]: Why did you remove the “of” here? I don’t understand

**Review by irvifa — CHANGES_REQUESTED** [review]

**Comment by irvifa** [comment]
@shahincsejnu both I and @kbhawkey had comments on your PR why dis you resolve that?

**Review by irvifa — COMMENTED** [review]
Thanks!
/lgtm
/approve

**Pull request merged**